specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualNetworkGateway.json

```
@@ -982,6 +982,129 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/startpacketcapture": {
```

```suggestion
    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/startPacketCapture": {
```
Can we use camelCase for endpoint name?